src/backend/aspen/api/views/phylo_runs.py

```diff
@@ -102,16 +111,18 @@ async def kick_off_phylo_run(
     for sample in user_visible_samples:
         pathogen_genomes.append(sample.uploaded_pathogen_genome)
 
-    # 4B - AlignedGisaidDump
-    aligned_gisaid_dump_query = (  # type: ignore
-        sa.select(AlignedGisaidDump)  # type: ignore
-        .join(AlignedGisaidDump.producing_workflow)  # type: ignore
+    # 4B - AlignedRepositoryData
+    aligned_repo_data_query = (  # type: ignore
+        sa.select(AlignedRepositoryData)  # type: ignore
+        .join(AlignedRepositoryData.producing_workflow)  # type: ignore
+        .where(AlignedRepositoryData.pathogen == pathogen)  # type: ignore
+        .where(AlignedRepositoryData.public_repository == public_repository)  # type: ignore
```

Comment:
This is totally just an aside, but I learned today that `.where()` also accepts multiple conditions as comma-separated arguments; they get treated as parts of `AND` clause (functionally the same as what you wrote here). Pretty cool.